### PR TITLE
ja: Update faq/extend-webpack.md

### PR DESCRIPTION
**Pull request by wanshot**
@inouetakuya @potato4d @aytdm
I updated faq/extend-webpack.md

Sorry I'm late. Please review it 🙏

resolve vuejs-jp#194

### CODE REVIEW

File: ja/faq/extend-webpack.md [hunk 1]
```diff
@@ -5,14 +5,26 @@ description: Webpack の設定を拡張するには？
...
      extend (config, { isDev, isClient }) {
-       // ...
+       // ..
+       config.module.rules.push(
+          {
+            test: /\.(ttf|eot|svg|woff(2)?)(\?[a-z0-9=&.]+)?$/,
+            loader: 'file-loader',
+          }
+        )
+        // isDev が true の場合、開発モードに設定します
```
Comment (aytdm) [inline ja/faq/extend-webpack.md]:
```suggestion
        // isDev が true の場合、webpack を開発モードに設定します
```
It might be a good idea to translate `webpack's `.

Comment (wanshot) [reply ja/faq/extend-webpack.md]: I reflected the correction. thx 👍

**Comment by aytdm** [comment]
@clarkdo cc: @inouetakuya 
I'm sorry for bothering you again, but could you please merge this PR? It seems like CircleCI works fine, however I have no authority to merge.

**Comment by inouetakuya** [comment]
I have no authority to merge this PR same as @aytdm  😢

**Pull request merged**